pkgs/test/nixpkgs-check-by-name/src/eval.rs

```diff
@@ -59,7 +59,7 @@ pub fn check_values<W: io::Write>(
         .args(["--arg", "attrsPath"])
         .arg(attrs_file.path())
```

Comment:
This one needs to be canonicalised aswell.

```suggestion
        .arg(attrs_file.path().canonicalize()?)
```

Once you do that, it appears to work on Darwin.

Perhaps it'd be better to just do that up top in the `attrs_file` declaration though?

Reply:
Good idea, done now!